core/src/test/java/org/mskcc/cbio/portal/util/TestStructuralVariantUtil.java

```diff
@@ -83,14 +80,52 @@ public void testValidStructuralVariantRecordMissingTranscriptInfo() {
         // record has site 1 entrez id and missing site 1 hugo symbol
         // and missing site 2 entrez id but present site 2 hugo symbol
         StructuralVariant record = new StructuralVariant();
+        record.setSampleId("SMP0001");
```

Comment:
this also seems to be testing the same case (no site1 gene .. site2 only has hugo)